assets/scss/_search.scss

```diff
@@ -14,7 +14,7 @@
         border-radius: 1rem;
     }
 
-    font-family: "Font Awesome 5 Free", $font-family-base;
+    font-family: "Font Awesome 6 Free", $font-family-base;
```

Comment:
This is now causing problems because it seems that the FA6 font contains regular character glyphs too:

> <img width="264" alt="image" src="https://user-images.githubusercontent.com/4140793/184670940-5c30626c-ab92-435e-a71f-9fabd02cf01c.png">

As compared to:

> <img width="227" alt="image" src="https://user-images.githubusercontent.com/4140793/184671035-64bf3371-4a8e-4fc4-961a-f67e287b8326.png">

The problem was there before this PR, so I'm ok merging this as is and fixing this problem after.

Reply:
Will follow up via:

- #1169